icon.schema.json

```diff
@@ -49,8 +74,27 @@
       "uniqueItems": true
     },
     "deprecated": {
-      "type": "boolean",
-      "default": false
+      "const": true
+    },
+    "deprecationReason": {
+      "$ref": "#/$defs/deprecationReasons"
+    },
+    "toBeRemovedInVersion": {
+      "$ref": "#/$defs/versionNumber",
+      "description": "The version this icon will be removed in."
+    }
+  },
+  "dependentRequired": {
+    "deprecated": ["deprecationReason", "toBeRemovedInVersion"]
+  },
+  "$defs": {
+    "deprecationReasons": {
+      "type": "string",
+      "enum": ["icon.brand", "alias.typo", "alias.name"]
```

Comment:
```suggestion
      "enum": ["icon.guideline-violation.brand", "alias.typo", "alias.guideline-violation.number"]
```

Reply:
And you should not be able to do `alias[0].depracationReason: "icon.bla"`, only "alias.bla" should be allowed, they should be seperated imo.

Reply:
Done.